spec/fabricators/bgeigie_import_fabricator.rb

```diff
@@ -1,4 +1,4 @@
 Fabricator(:bgeigie_import) do
-  source File.new(Rails.root.join("spec/fixtures/bgeigie#{Fabricate.sequence}.log"))
+  source { sequence(:source) { |n| Rails.root.join("spec/fixtures/bgeigie#{n % 6}.log").open(File::RDONLY) } }
```

Comment:
The `RDONLY` is interesting. Is it just for safety or was there something in the test that called for it?